src/rabbit_stomp_processor.erl

```diff
@@ -493,6 +495,17 @@ cancel_subscription({ok, ConsumerTag, Description}, Frame,
             end
     end.
 
+delete_binding(Channel, #subscription{exclusive = Exclusive, queue = Queue, exchange = Exchange, routing_key = RoutingKey}) ->
+    case Exclusive of
+        true ->
```

Comment:
Exclusive queue allow connect only from one connection. With not exclusive queue we must share bindings information between other connections (erlang processes) that add performance overhead.